diff --git a/core/pathmap.mk b/core/pathmap.mk
index 84d2b9d..4eb6b80 100644
--- a/core/pathmap.mk
+++ b/core/pathmap.mk
@@ -113,6 +113,7 @@ FRAMEWORKS_BASE_SUBDIRS := \
 	    media/mca/filterpacks \
 	    drm \
 	    opengl \
+	    privacy \
 	    sax \
 	    telecomm \
 	    telephony \
diff --git a/target/product/core_minimal.mk b/target/product/core_minimal.mk
index f221094..b6ad632 100644
--- a/target/product/core_minimal.mk
+++ b/target/product/core_minimal.mk
@@ -29,6 +29,7 @@ PRODUCT_PACKAGES += \
     HTMLViewer \
     MediaProvider \
     PackageInstaller \
+    PDroid \
     SettingsProvider \
     Shell \
     bcc \
